components/hab/src/cli.rs

```diff
@@ -810,6 +810,8 @@ pub fn sub_sup_run() -> App<'static, 'static> {
             "The listen address for the gossip system [default: 0.0.0.0:9638]")
         (@arg LISTEN_HTTP: --("listen-http") +takes_value {valid_socket_addr}
             "The listen address for the HTTP Gateway [default: 0.0.0.0:9631]")
+        (@arg HTTP_DISABLE: --("http-disable") -D
+            "Disable the HTTP Gateway completely [default: enabled]")
```

Comment:
It might be a bit clearer to write `default: false`, since `enabled` could either refer to the HTTP gateway or the `--http-disable` flag.

Alternatively, you could make this an *option* that takes a boolean rather than a flag:
```
        --http-gateway <BOOL> [default: true]
```

@apriofrost: what do you think?

Reply:
As do I - I'll change it.